### PR TITLE
add r2dii.data

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
@conda-forge/core Hello, anybody to approve this PR please ? Need this for #16757 and #16758

File: recipes/r-r2dii.data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('r2dii.data')"           # [not win]
+    - "\"%R%\" -e \"library('r2dii.data')\""  # [win]
+
+about:
+  home: https://2degreesinvesting.github.io/r2dii.data/, https://github.com/2DegreesInvesting/r2dii.data
+  license: CC0
```
Comment (dbast) [inline recipes/r-r2dii.data/meta.yaml:39]:
```suggestion
  license: CC0-1.0
  license_file: LICENSE
```

File: recipes/r-r2dii.data/LICENSE [hunk 2]
```diff
@@ -0,0 +1 @@
...ther case contrary to Affirmer's express Statement of Purpose.  Limitations and Disclaimers.  a. No trademark or patent rights held by Affirmer are waived, abandoned, surrendered, licensed or otherwise affected by this document.  b. Affirmer offers the Work as-is and makes no representations or warranties of any kind concerning the Work, express, implied, statutory or otherwise, including without limitation warranties of title, merchantability, fitness for a particular purpose, non infringement, or the absence of latent or other defects, accuracy, or the present or absence of errors, whether or not discoverable, all to the greatest extent permissible under applicable law.  c. Affirmer disclaims responsibility for clearing rights of other persons that may apply to the Work or any use thereof, including without limitation any person's Copyright and Related Rights in the Work. Further, Affirmer disclaims responsibility for obtaining any necessary consents, permissions or other rights required for any use of the Work.  d. Affirmer understands and acknowledges that Creative Commons is not a party to this document and has no duty or obligation with respect to this CC0 or use of the Work.
```
Comment (dbast) [inline recipes/r-r2dii.data/LICENSE:1]: This should be a copy of this? https://github.com/2DegreesInvesting/r2dii.data/blob/main/LICENSE.md

Comment (FernandezMathieu) [reply recipes/r-r2dii.data/LICENSE:1]: Yes but minified. Text must be formatted ?

**Comment by dbast** [comment]
@FernandezMathieu thanks!

**Pull request merged**